library/res/color-v21/default_tab_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:color="?android:textColorPrimary" android:state_selected="true" />
```

Comment:
Yes, you're right, it's available. Doesn't look like it can be used as a color attribute though, weirdly - it doesn't resolve to the correct color (the resource ID gets parsed as the color, rather than resolving it). Using it as a drawable resource, like for a background, seems to work. So confused.